alerting/src/main/kotlin/com/amazon/opendistroforelasticsearch/alerting/action/GetAlertsRequest.kt

```diff
@@ -20,32 +20,39 @@ import org.elasticsearch.action.ActionRequest
 import org.elasticsearch.action.ActionRequestValidationException
 import org.elasticsearch.common.io.stream.StreamInput
 import org.elasticsearch.common.io.stream.StreamOutput
+import org.elasticsearch.index.query.TermsQueryBuilder
 import java.io.IOException
 
 class GetAlertsRequest : ActionRequest {
     val table: Table
     val severityLevel: String
     val alertState: String
     val monitorId: String?
```

Comment:
Lets remove these attributes since they are used for filtering the query. These query filters can be added to the attribute `filter` and we can change `filter` to be type: `List<TermsQueryBuilder>`